docs/ReleaseNotes.md

```diff
@@ -1,6 +1,16 @@
 Release Notes for Version 2
 ============================
 
+Build 011
+-------
+
+Published as version 2.8.0
```

Comment:
For semantic versioning, it should be 2.7.1. The 0 changes to 1 for bug fixes, the 7 changes to 8 when new features are added, and 2 turns to 3 when backwards compatibility is broken and the caller must do something different than before.

Reply:
I fixed it.